sdk/keyvault/azsecrets/autorest.md

```diff
@@ -8,12 +8,17 @@ These settings apply only when `--go` is specified on the command line.
 ``` yaml
 go: true
 version: "^3.0.0"
-input-file: https://github.com/Azure/azure-rest-api-specs/blob/1e2c9f3ec93078da8078389941531359e274f32a/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.2/secrets.json
+input-file:
+- https://github.com/Azure/azure-rest-api-specs/blob/ecdce42924ed0f7e60a32c74bc0eb674ca6d4aae/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.2/common.json
+- https://github.com/Azure/azure-rest-api-specs/blob/ecdce42924ed0f7e60a32c74bc0eb674ca6d4aae/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.2/secrets.json
+- https://github.com/Azure/azure-rest-api-specs/blob/ecdce42924ed0f7e60a32c74bc0eb674ca6d4aae/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.2/rbac.json
+- https://github.com/Azure/azure-rest-api-specs/blob/ecdce42924ed0f7e60a32c74bc0eb674ca6d4aae/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.2/securitydomain.json
```

Comment:
Aren't security domains relevant only to MHSM?

Reply:
Yes but these are the files I found I needed to have all of the models for the generated code. If there's a different combination you know of I'll remove them in a separate PR